src/core/opamSystem.ml

```diff
@@ -1633,6 +1633,31 @@ let translate_patch ~dir orig corrected =
   end;
   close_in ch
 
+let patch_cmd = lazy begin
+  match OpamStd.Sys.os () with
+  | DragonFly
+  | FreeBSD
+  | NetBSD
+  | OpenBSD -> "gpatch"
+  | Cygwin
+  | Darwin
```

Comment:
mmmh, this is actually tricky. Macports provides `gpatch` but Homebrew provides `patch`

Reply:
Fixed by resolving both